examples/default.yaml

```
@@ -456,8 +456,8 @@ hostResolver:
   # predefined to specify the gateway address to the host.
   # 🟢 Builtin default: null
   hosts:
-    # guest.name: 127.1.1.1
-    # host.name: host.lima.internal
+  # guest.name: 127.1.1.1
+  # host.name: host.lima.internal
```

I think it looks better to maintain the indentation to show that the lines are commented out key/value mappings. Just move the `#` to the front:

```suggestion
  #   guest.name: 127.1.1.1
  #   host.name: host.lima.internal
```